recipe/meta.yaml

```diff
@@ -1,106 +1,279 @@
+      number: {{ build_number }}        # [gpu_variant == "none" and x86_64_opt == "none"]
+      string: cuda{{ cuda_compiler_version | replace('.', '') }}_v3_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [(gpu_variant or "").startswith('cuda') and (x86_64_opt == "v3")]
+      string: cuda{{ cuda_compiler_version | replace('.', '') }}_v2_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [(gpu_variant or "").startswith('cuda') and (x86_64_opt == "v2")]
+      string: cuda{{ cuda_compiler_version | replace('.', '') }}_v1_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [(gpu_variant or "").startswith('cuda') and (x86_64_opt == "none")]
+      string: cpu_v3_{{ blas_impl }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                              # [gpu_variant == "none" and x86_64_opt == "v3"]
+      string: cpu_v2_{{ blas_impl }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                              # [gpu_variant == "none" and x86_64_opt == "v2"]
+      string: cpu_v1_{{ blas_impl }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                              # [gpu_variant == "none" and x86_64_opt == "none"]
+      string: mps_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}                                                 # [gpu_variant == "metal"]
+      missing_dso_whitelist:                                                                         # [s390x or (gpu_variant or "").startswith('cuda')]
+        - "**/libcuda.so*"                                                                           # [(gpu_variant or "").startswith('cuda')]
+        - '$RPATH/ld64.so.1'  # [s390x]
+      
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - {{ compiler('cuda') }}                              # [(gpu_variant or "").startswith('cuda') and (gpu_variant != "cuda-11")]
+        - cmake
+        - ninja-base
+        - pkgconfig
+        - patch  # [osx or (unix and blas_impl == "mkl") or aarch64]
+        - m2-patch  # [win]
+
+      host:
+        - cudatoolkit      {{ cuda_compiler_version }}        # [gpu_variant == "cuda-11"]
+        - cuda-version     {{ cuda_compiler_version }}        # [(gpu_variant or "").startswith('cuda')]
+        - cuda-cudart-dev  {{ cuda_compiler_version }}        # [(gpu_variant or "").startswith('cuda') and (gpu_variant != "cuda-11")]
+        - libcublas-dev    {{ cuda_compiler_version }}        # [(gpu_variant or "").startswith('cuda') and (gpu_variant != "cuda-11")]
+        - cuda-compat      {{ cuda_compiler_version }}        # [(gpu_variant or "").startswith('cuda') and (gpu_variant != "cuda-11") and linux]
+        - openblas-devel {{ openblas }}                       # [(not (gpu_variant or "").startswith('cuda')) and blas_impl == "openblas"]
+        - mkl-devel {{ mkl }}                                 # [(not (gpu_variant or "").startswith('cuda')) and blas_impl == "mkl"]
+        - intel-openmp {{ mkl }}                              # [(not (gpu_variant or "").startswith('cuda')) and blas_impl == "mkl"]
```

Comment:
Seeing in the logs:
`                   INFO   WARNING (llama.cpp): dso library package defaults/linux-64::intel-openmp==2023.1.0=hdb19cb5_46306 in requirements/run but it is not used `
The compiler openmp seems to be used instead of this one.

Reply:
@cbouss I didn't change that from the original deps for llama.cpp - let me re-run and see if I can remove that.